lib/java/change-directory/src/main/java/org/enso/change/directory/WorkingDirectories.java

```diff
@@ -0,0 +1,17 @@
+package org.enso.change.directory;
+
+import org.enso.common.Platform;
+
+public final class WorkingDirectories {
+  private WorkingDirectories() {}
+
+  private static final WorkingDirectory CURRENT =
+      switch (Platform.getOperatingSystem()) {
+        case LINUX, MACOS -> PosixWorkingDirectory.INSTANCE;
+        case WINDOWS -> WindowsWorkingDirectory.INSTANCE;
+      };
+
+  public static WorkingDirectory getCurrent() {
```

Comment:
To minimize the conceptual surface:
- move this method to `WorkingDirectory`
- make this class package private
- or rather merge this into `WorkingDirectory` abstract class.

Reply:
Done in c01d17d35f59f9253d6c063a70253974e856381c